src/generated/mistica-icons/icons-keywords.tsx

```diff
@@ -381,15 +381,15 @@ const iconKeywords: {[key: string]: Array<string>} = {
     ],
     audio: ['sound', 'audio signal', 'soundwave', 'sonido', 'sinal de áudio', 'Ton'],
     aura: ['assistant', 'virtual', 'helper', 'aura', 'aura virtual', 'Helfer'],
-    'authentication-failure': [
+    'autentication-failure': [
```

Comment:
Unrelated change. It's just an update in Mistica design repo

Reply:
but it's a typo, isn't it?

Reply:
we added to 15.0.0 major version to solve this typo in some icons and avoid breaking changes now
https://github.com/Telefonica/mistica-web/issues/626